packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -0,0 +1,339 @@
+    background-color: var(--brand-color-canvas-default);
+  }
+
+  .AnchorNav--stuck {
+    background-color: var(--brand-color-canvas-default);
+  }
+  .AnchorNav-inner-container {
+    margin: 0 auto;
+    display: flex;
+  }
+}
+
+/*
+ * 3. AnchorNav links
+ */
+.AnchorNav-link {
+  display: flex;
+  width: calc(100% - 80px);
+  text-decoration: none;
+  height: 48px;
+  align-items: center;
+  justify-items: flex-start;
+}
+
+.AnchorNav--expanded .AnchorNav-link {
+  margin: 0 0 2px; /* allow space for focus border */
+  animation: fade-in 1s;
+  animation-timing-function: cubic-bezier(0.33, 1, 0.68, 1); /* TODO: replace with global token for animation timings */
+}
+
+/* Up to large breakpoint */
+@media screen and (max-width: 63.25rem) {
+  .AnchorNav-link {
+    position: absolute;
+    left: 0;
+  }
+}
+
+/* Large breakpoint and up */
+@media screen and (min-width: 63.25rem) {
+  .AnchorNav-link {
+    position: relative;
+    margin: 0 1px; /* to provide space for focus border */
+  }
+
+  .AnchorNav-link--center {
+    justify-content: center;
+  }
+
+  .AnchorNav-link--start {
+    justify-content: flex-start;
+    width: auto;
+    margin-right: var(--base-size-48);
+  }
+
+  .AnchorNav-link::after {
+    content: '';
+    position: absolute;
+    bottom: 10px;
+    left: 0;
+    width: 100%;
+    height: 2px;
+    background-color: var(--brand-AnchorNav-border-nav-default);
+    opacity: 0;
+    transition: opacity 300ms, transform 300ms;
+    opacity: 1;
+    transform: scale(0);
+    transform-origin: center;
+  }
+
+  .AnchorNav-link:hover::after,
+  .AnchorNav-link:focus::after,
+  .AnchorNav-link[data-active='true']::after {
+    opacity: 1;
+    transform: translate3d(0, 0.2em, 0);
+    transform: scale(0.4);
+  }
+
+  .AnchorNav-link:active::after {
+    background-color: var(--brand-AnchorNav-border-nav-pressed);
+    transform: scale(0.6);
+  }
+
+  .AnchorNav-link[data-active='true']::after {
+    background-color: var(--brand-AnchorNav-border-nav-pressed);
+    opacity: 1;
+  }
+
+  .AnchorNav-link:focus-visible::after {
+    opacity: 0;
+  }
+}
+
+/*
+ * 4. AnchorNav link labels
+ */
+.AnchorNav-link-label {
+  font-weight: var(--brand-heading-weight-400);
+  font-size: var(--brand-text-size-300);
+  line-height: var(--brand-text-lineHeight-300);
+  font-family: var(--brand-heading-fontFamily);
+  letter-spacing: var(--brand-heading-letterSpacing-400);
+  color: var(--brand-color-text-default);
+  transition: color cubic-bezier(0.33, 1, 0.68, 1) 0.2s; /* TODO: replace with global token for animation timings */
+  white-space: nowrap;
+}
+
+.AnchorNav--with-default-background-color .AnchorNav-link-label {
+  background-color: var(--brand-color-canvas-default);
+}
+
+/* Up to large breakpoint */
+@media screen and (max-width: 63.25rem) {
+  .AnchorNav-link-label {
+    background-color: var(--brand-color-canvas-default);
+  }
+
+  .AnchorNav--expanded .AnchorNav-link-label--is-active {
+    color: var(--brand-color-focus);
```

Comment:
<img width="194" alt="AnchorNav-link active state showing a line indicator" src="https://user-images.githubusercontent.com/175638/206260711-211307fa-3c27-4e50-863b-f8aa4041bbb1.png">

Let's update this style to use the line indicator to align with the active state on larger viewports.

Reply:
@danielguillan I've implemented this now.

I've kept the blue indicator in for now, as it's hard to notice the active line as you're scrolling the page otherwise. I feel we need both. Let me know what you think. Happy to revise in a follow-up PR.

Without active text color:

https://user-images.githubusercontent.com/13340707/206477522-949a5d49-0d06-45ce-88e4-ffed20662eec.mov

With active text color:

https://user-images.githubusercontent.com/13340707/206477747-974e9971-dacd-468b-8bad-271d803502bb.mov